src/frontend/sharedComponents/ErrorBottomSheet.tsx

```diff
@@ -74,8 +76,9 @@ export const ErrorBottomSheet = (props: ErrorModalProps) => {
       onDismiss={closeSheet}
       isOpen={isOpen}>
       <BottomSheetModalContent
-        icon={<LogoWithErrorIcon />}
-        title={formatMessage(m.somethingWrong)}
+        icon={<ErrorIcon style={{marginTop: 80}} />}
+        title={formatMessage(title || m.somethingWrong)}
+        description={description ? formatMessage(description) : undefined}
```

Comment:
It looks like from the other Issue the description is supposed to be 16 pixels. Might need to set up a ternary for the styles here as well.

Reply:
I updated the font size (as well as the icon size to match the design system):
<img width="292" alt="image" src="https://github.com/user-attachments/assets/8e528d26-eb49-4c4d-aec6-e9e8c4acb0d0">

https://github.com/digidem/comapeo-mobile/pull/771/commits/077e8ab254ae86f503796c235e2cee57d48a957a